remove-wheels/action.yml

```diff
@@ -0,0 +1,50 @@
+name: Scientific Python / Remove Old Wheels
+description: A GitHub Action to remove old wheels
+permissions:
+  actions: read
+  contents: read
+  metadata: read
+author: "Scientific-Python"
+# TODO: have to think about versioning; whether to version separately, or
+# for it to be in sync with the version for the upload action
+version: "0.1.0"
+
+inputs:
+  n_latest_uploads:
+    description: 'The number of previous wheel uploads to keep'
+    required: false
+    default: '5'
+  anaconda_nightly_upload_organization:
+    description: 'Anaconda Cloud organisation name to remove the wheels from'
+    required: false
+    default: scientific-python-nightly-wheels
+  anaconda_nightly_token:
+    description: 'Anaconda Cloud API token to authenticate with'
+    required: true
+
+# TODO: Linux only for now, need to see how to add macOS support
+runs:
+  using: "composite"
+  steps:
+    - name: Set up pixi
+      uses: prefix-dev/setup-pixi@ba3bb36eb2066252b2363392b7739741bb777659  # v0.8.1
+      with:
+        locked: true
+        cache: true
+        cache-write: ${{ github.event_name == 'push' && github.ref_name == 'main' }}
+        # Avoid post cleanup errors if action run multiple times
+        post-cleanup: false
+
+    - name: Install dependencies
+      shell: bash
+      run: |
+        sudo apt-get update && sudo apt-get install -y curl jq
```

Comment:
Is there a nicer way to add macOS support instead of having to do something like

```yaml
if: runner.os == 'macOS'
run: brew install curl jq # and so on
```

I don't know too much about how `pixi` works, yet – I probably should 😅

Reply:
I added `jq` to the dependencies in `pixi.toml` and removed this step, I guess that should work.